utils/language.ts

```diff
@@ -1,13 +1,13 @@
 export function matchLanguages(languages: string[], acceptLanguages: readonly string[]): string | null {
   {
-    // const lang = acceptLanguages.map(userLang => languages.find(lang => lang.startsWith(userLang))).filter(v => !!v)[0]
-    // TODO: Support es-419, remove this code if we include spanish country variants
-    const lang = acceptLanguages.map(userLang => languages.find((lang) => {
-      if (userLang.startsWith('es-') && userLang !== 'es-ES')
-        return lang === 'es-419'
+    const lang = acceptLanguages.map(userLang => languages.find(lang => lang.startsWith(userLang))).filter(v => !!v)[0]
```

Comment:
This should be reverted: since `es-419` is not official, maybe missing from browser preferences.

This change was to detect any lang from Latinoamérica: for example, I included `es-AR` at top on my chrome lang preferences, and so, when entering to the app, Elk will pick `es-AR` as `es-419`:

`chrome://settings/?search=idiomas`

![imagen](https://user-images.githubusercontent.com/6311119/215736881-1dfef242-e59f-4729-9d83-006ee3abe41c.png)

Reply:
I left the TODO comments in the module for when all the variants were included, not `es-419`

Reply:
I might misunderstood this instruction. Reverted.